wazuh/wazuh-kibana-app
# Adapt the new global menu redirections

### Description

Fixed redirections to adapt to the new global menu
 
### Issues Resolved

#5820 

### Evidence

Endpoint Summary
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/f22e77ca-8b08-4302-8b79-195a9dc6505a)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/6af1c24a-3e36-40f8-b895-575838a09dea)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/ad22f47b-1b1e-4fc6-94ad-097f239723c4)

Groups
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/6621ee7c-771d-4fa8-bd19-3b7026db6b22)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/becb2dc9-2a43-49fd-897c-ca6ba18db752)

Rules


### Test

- Set up the environment,
- Navigate to: 
  - Endpoint Summary
  - Groups
  - Rules
  - Cluster
  - Statistics
- Check that links work as expected.

### Check List
